src/utils/viewer/viewer.cpp

```diff
@@ -540,21 +498,11 @@ void Viewer::drawEvent() {
 
     // rendering
     esp::gfx::RenderCamera::Flags flags;
-    if (frustumCullingEnabled_) {
+    if (simulator_->isFrustumCullingEnabled()) {
       flags |= esp::gfx::RenderCamera::Flag::FrustumCulling;
     }
     renderCamera_->draw(objectPickingHelper_->getDrawables(), flags);
 
-    // Neither the blend equation, nor the blend function is changed,
```

Comment:
Suggest to keep this comments. I intend to put it here, since this is some valuable lesson I learned when debugging the "blending" issues. It cost me a couple of hours to figure it out even though it is not used eventually (since I found some other simpler solutions).